### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by edubew**
# Window popup
> In this milestone I did the following:
-  Installed html, css and javascript linters
- Updated the main page so that the projects section is created dynamically
- Created the popup window and added it to the main page with javascript
- Added styling to the popup window for both the mobile and desktop version
- Added functionality such that when the "see projects" button is clicked, the popup window displays, and when the close button is clicked, the popup window disappears

### CODE REVIEW

File: popup.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+.modal-window {
+  background: hsl(242deg 38% 27%);
+  border: 1px solid #fff;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  margin: 50px 24px 0 24px;
+  display: flex;
+  position: fixed;
+  z-index: 2;
+  top: -36px;
+  overflow: hidden;
```
Comment (Ismailco) [inline popup.css]: - [ ] Good job implementing the popup however, the user can't scroll down in the popup, please consider fixing this issue

Screenshot_20220512_201356

File: popup.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
+// Instantiate projects object
+
+const projects = [
+  {
```
Comment (Ismailco) [inline popup.js:4]: - [x] Each object should contain at least the following information about the project: ```name```, ```description```, ```featured image```, ```technologies```, ```link to live version```, and ```link to the project source code```, you can add the links and change ```title``` to ```name``` so your code meets the requirements

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @edubew,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally
✔️ Follow best practices

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: popup.css (hunk 1 above)
Comment (edubew) [reply popup.css]: Hello @Ismailco ,
Thank you for the review. 
I will really appreciate it if you could provide some more insight on how I can add the scroll for the modal. 
> I have tried the following:
   - Adding an "overflow-y: "visible" to the modal container class in css file
   - Adding classname.style.overflowY="auto" in the javascript file
   
But neither of the two methods are working

**Comment by edubew** [comment]
Hello @Ismailco
Thank you for the review. 
I will really appreciate it if you could provide some more insight on how I can add the scroll for the modal. 
> I have tried the following:
   - Adding an "overflow-y: "visible" to the modal container class in css file
   - Adding classname.style.overflowY="auto" in the javascript file
   
But neither of the two methods are working

**Pull request merged**